warehouse/macaroons/models.py

```diff
@@ -69,6 +69,10 @@ class Macaroon(db.Model):
     # body of the permissions ("V1") caveat.
     permissions_caveat = Column(JSONB, nullable=False, server_default=sql.text("'{}'"))
 
+    # OIDC claims associated with this macaroon. Should only exist for OIDC
+    # publisher macaroons.
```

Comment:
We'll probably want to use this column for other things in the future (e.g. collapsing `permissions_caveat` into it), so let's tweak this:

```suggestion
    # Additional state associated with this macaroon.
    # For OIDC publisher-issued macaroons, this will contain a subset of OIDC claims.
```